Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Utils/FrameCaptureBus.h

```diff
@@ -43,45 +47,60 @@ namespace AZ
             //! Capture final screen output for the specified window and save it to given file path.
             //! The image format is determinate by file extension
             //! Current supported formats include ppm and dds. 
-            //! @param filepath The output file path. 
+            //! @param imagePath The output file path. 
             //! @param windowHandle The handle to the AzFrameWork::NativeWindow that is being captured
             //! @return value is the frame capture Id, on failure it will return InvalidFrameCaptureId
-            virtual FrameCaptureId CaptureScreenshotForWindow(const AZStd::string& filePath, AzFramework::NativeWindowHandle windowHandle) = 0;
+            virtual FrameCaptureOutcome CaptureScreenshotForWindow(const AZStd::string& imagePath, AzFramework::NativeWindowHandle windowHandle) = 0;
```

Comment:
Please update the version field in `Atom/Feature/Common/gem.json` if these API changes are breaking (update the major version to 1.0.0) or new feature (minor version to 0.1.0)

Reply:
Thanks for pointing that out. Updated.